resource-aggregate/cqrs/aggregate/aggregate.go

```diff
@@ -0,0 +1,243 @@
+package aggregate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/plgd-dev/cloud/resource-aggregate/cqrs/eventstore"
+)
+
+// Command user defined command that will handled in AggregateModel.HandleCommand
+type Command = interface{}
+
+// AggregateModel user model for aggregate need to satisfy this interface.
+type AggregateModel = interface {
+	eventstore.Model
+
+	HandleCommand(ctx context.Context, cmd Command, newVersion uint64) ([]eventstore.Event, error)
+	TakeSnapshot(version uint64) (snapshotEvent eventstore.Event, ok bool)
+	GroupId() string //defines group where model belows
```

Comment:
We don't have groups, they were removed. Or what does this group represents?

Reply:
GroupID = DeviceID so we can renamed them to `resourceServerID`